src/styles/partials/components/_footer.scss

```diff
@@ -0,0 +1,94 @@
+/* Add css to test in this file. It should be cleaned up every pr, as the css should make into the components project.*/
+.dg_county-footer {
+  border-top: $border-width-largest solid $red;
+}
+.dg_secondary-footer {
+  background-color: $gray-dark;
+  padding: $padding-large;
+}
+.dg_copyright {
+  color: $white;
+  font-family: $default-heading-font;
+  font-size: $small-heading-font-size;
+  text-transform: uppercase;
+}
+.dg_download-blurb {
+  font-style: italic;
+}
+.dg_logo-sm-txt {
+  color: $gray;
+  display: block;
+  font-family: $default-heading-font;
+  font-size: $smallest-font-size;
+  font-weight: $font-bold;
+  letter-spacing: 10px;
+  text-transform: uppercase;
+}
+.dg_logo-h1 {
+  color: $black;
+  font-family: $default-heading-font;
+  font-size: $small-heading-font-size;
+  font-weight: $font-bold;
+  text-transform: uppercase;
+}
+.dg_brand-footer {
+  margin-bottom: $default-margin;
+  margin-top: $margin-largest;
+
+  .img-fluid {
+    margin-bottom: $margin-small;
+  }
+
+  p {
+    padding-bottom: $padding-largest;
+  }
+
+  a {
+    text-decoration: none;
+  }
+  .dg_alt-list {
```

Comment:
I'm a little nervous of what this is doing, gonan pull up the prototype and check this out. ideally we wouldnt' be doign anything to this list.